### PR TITLE
Tutorial clean-up

### PR DESCRIPTION
**Pull request by RenateEilers**
This PR updates some outdated links and filenames in the documentation and tutorials.

### CODE REVIEW

**Review by jasagredo — DISMISSED** [review]
Thanks!!

**Comment by jasagredo** [comment]
May you rebase (locally) instead of merging `main` into this branch? You can then resign with
```
git rebase --exec 'git commit --amend --no-edit -n -S' -i HEAD~N
```
Where `N` is the number of commits you want to sign

File: docs/website/contents/for-developers/ChainSync.md [hunk 1]
```diff
@@ -148,3 +155,5 @@ trim their fragment so that it is anchored at our anchor point.
 - We could, if needed, bind this more tightly still; as long as we have a
   sufficient number of headers to determine if their chain is preferred over
   ours (this needs careful consideration when adopting genesis).
+
+[^glossary]: See the [Glossary][]
```
Comment (jasagredo) [inline docs/website/contents/for-developers/ChainSync.md:159]: Footnote missing link?

File: docs/website/contents/for-developers/HardWonWisdom.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ Step two will be to occasionally extract cohesive insights from this bag, creati
 
 One of the `HardForkBlock` combinator's major responsibilities is providing an `EpochInfo` to the ledger code.
 This `EpochInfo` uses the `Either` monad to return `Right` only when the query can be answered with certainty.
-For more information on when that is, see the Consensus Report and the recordings of Edsko's presentations at the Weekly IOG Seminar.
+For more information on when that is, see the [Consensus Report](https://ouroboros-consensus.cardano.intersectmbo.org/pdfs/report.pdf) and the recordings of Edsko's presentations at the Weekly IOG Seminar([1](https://drive.google.com/file/d/1m_jKQM_gxBm0ctLqIq9NGj5_nXPI66Su/view),[2](https://drive.google.com/file/d/1QIJ-VBlj-txB6K6E7DIEnY5TzaD89qQm/view)).
```
Comment (jasagredo) [inline docs/website/contents/for-developers/HardWonWisdom.md:11]: The links for these talks are not publicly accessible.

File: docs/website/contents/for-developers/ChainSync.md [hunk 3]
```diff
@@ -1,10 +1,12 @@
 # Chain sync client specification
 
+The aim of the chain sync client is to replicate the chain of an upstream node. For every upstream node that the local node is connected to there is a separate instance of the client. In the following, we consider a single instance.
```
Comment (jasagredo) [inline docs/website/contents/for-developers/ChainSync.md:3]: Line wrapping at ~80 chars?

Comment (jasagredo) [reply docs/website/contents/for-developers/ChainSync.md:3]: Not sure if we strictly follow that though

**Review by jasagredo — APPROVED** [review]

File: docs/website/contents/for-developers/HardWonWisdom.md (hunk 2 above)
Comment (amesgen) [reply docs/website/contents/for-developers/HardWonWisdom.md:11]: I think it is still convenient to link to them (also, there was internal talk on making (most of) these publicly available at *some* point), especially as HardWonWisdom is not supposed to be polished documentation.

File: docs/website/contents/for-developers/ChainSync.md (hunk 3 above)
Comment (dnadales) [reply docs/website/contents/for-developers/ChainSync.md:3]: @nfrisby made a point for not wrapping at all when editing markdown files. I think he prefers one line per sentence.

Comment (jasagredo) [reply docs/website/contents/for-developers/ChainSync.md:3]: Let's resolve then

**Pull request merged**